mingw_smoketests.py

```diff
@@ -233,6 +233,13 @@ def test_c_ext_build(self):
         import textwrap
         from pathlib import Path
 
+        if Path(__file__) / 'python.exe' == Path(sys.executable):
```

Comment:
Is this the same as `not sysconfig.is_python_build()`?

Reply:
oh, there is a function for that? I will use that then

Reply:
In theory, I've never tried it personally.

Reply:
let's try it now